### DISCUSSION TITLE
OCR Predictor on Chinese Messages

### DISCUSSION THREAD

**Discussion by orlandito24** [mindee/doctr#1402] [Q&A]
I'm wondering if there is a way to use the ocr_predictor, but have it so it is able to interpret Chinese characters? Currently it is placing boxes over the Chinese characters, but its trying to print them as English characters.

**Comment by felixdittrich92** [comment]
Our current recognition models are pretrained on ~10M real samples (word crops) from french vocab.

For chinese you have to train your own model #1403
